### ISSUE TITLE
GI Bill Statement of Benefits does not properly alert users of an error 

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#56601]
## Issue Description
In a research session, I observed a screenreader user navigate to the Post 9/11 GI Bill Statement of Benefits page (va.gov/education/gi-bill/post-9-11/ch-33-benefit/status) while logged in. The page was empty except a heading containing an error message and a button to go back to the post-9/11 GI Bill page. There are few things that need to be fixed here:

1. The error message should follow the guidance on [error messages](https://design.va.gov/content-style-guide/error-messages/system) in the platform documentation. I'm not sure exactly what the problem was so can't say if it needs to follow the system or access recommendations, but all error messages should use an alert component. I believe the component has `role=alert` baked in, but if not, that should be added to so it's read to screenreaders appropriately.
2. There should be a heading on the page to give users context and help them know where they've landed (currently, the breadcrumb is doing that job)
3. Style the "back to" link as a link, instead of a button, if it is indeed an anchor tag. 

## Acceptance Criteria
- [ ] Page uses components according to the design system docs/accessibility best practices.

**Comment by joshkimux** [comment]
@artsymartha68 could we re-route this to the GI Bill team?

**Comment by andaleliz** [comment]
@mattpointzxer0 asked me to create this, but didn't respond yet with who it should be assigned to. Thanks for looking at it @joshkimux.[ There's a thread ](https://dsva.slack.com/archives/C02825JJJRW/p1680898112929519) where some other folks are jumping in too.

**Comment by joshkimux** [comment]
Perfect, I've tagged Martha for awareness into that thread too

**Comment by andaleliz** [comment]
@mattpointzxer0 I don't see Eddie Ricks as someone I can assign this to ([thread](https://dsva.slack.com/archives/CE4304QPK/p1681136917335529)) - assigning to you in the interim. Is there someone else I should assign this to so it can get prioritized? Thanks!

**Comment by andaleliz** [comment]
@mattpointzxer0 I completely dropped the ball on this.  Is there someone I can assign this to who can help with remediation?  I saw a functioning version of this page in a research session and spotted some other issues on the page that need attention.

**Comment by mattpointzxer0** [comment]
@andaleliz all good. This issue is a little stale, so I'm not 100% positive Eddie is still the right person, but I'd give him a try and at the least hopefully he can redirect you.
He, like most of the education team doesn't use our Slack instance, so you'll want to connect with him on Teams or via email.

**Comment by emcruer** [comment]
IIR has worked to define the errors (as far as it's possible) and then with CAIA to update messaging. It's in the process of being corrected.
